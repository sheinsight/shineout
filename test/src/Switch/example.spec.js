import exampleTest from '../../example'

exampleTest('Switch')
