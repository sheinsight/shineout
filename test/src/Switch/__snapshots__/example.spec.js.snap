// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Switch[snapshot] Snapshot test: Switch[site/pages/components/Switch/example-1-base.tsx] 1`] = `
<label>
  <span />
  <input />
  <i />
  <span />
</label>
`;

exports[`Switch[snapshot] Snapshot test: Switch[site/pages/components/Switch/example-2-disabled.tsx] 1`] = `
<div>
  <label>
    <span />
    <input />
    <i />
    <span />
  </label>
  <button>
    <span>
      Toggle
    </span>
  </button>
</div>
`;

exports[`Switch[snapshot] Snapshot test: Switch[site/pages/components/Switch/example-3-content.tsx] 1`] = `
<div>
  <label>
    <span>
      å¼€
    </span>
    <input />
    <i />
    <span />
  </label>
  <br />
  <label>
    <span>
      <i />
    </span>
    <input />
    <i />
    <span />
  </label>
</div>
`;

exports[`Switch[snapshot] Snapshot test: Switch[site/pages/components/Switch/example-4-size.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span />
  </label>
  <br />
  <label>
    <span />
    <input />
    <i />
    <span />
  </label>
  <br />
  <label>
    <span />
    <input />
    <i />
    <span />
  </label>
</div>
`;

exports[`Switch[snapshot] Snapshot test: Switch[site/pages/components/Switch/example-5-loading.tsx] 1`] = `
<div>
  <label>
    <input />
    <i />
    <span>
      <div />
    </span>
  </label>
  <label>
    <input />
    <i />
    <span>
      <div />
    </span>
  </label>
  <br />
  <label>
    <span />
    <input />
    <i />
    <span>
      <div />
    </span>
  </label>
  <label>
    <span />
    <input />
    <i />
    <span>
      <div />
    </span>
  </label>
  <br />
  <label>
    <span />
    <input />
    <i />
    <span>
      <div />
    </span>
  </label>
  <label>
    <span />
    <input />
    <i />
    <span>
      <div />
    </span>
  </label>
</div>
`;
